streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1360,7 +1395,10 @@ public int hashCode() {
         public Subtopology(final int id,
                     final Set<org.apache.kafka.streams.TopologyDescription.Node> nodes) {
```

Comment:
Not introduced by this PR, but please fix indention.

Reply:
ack